salesforce-gui/src/test/java/org/fundacionjala/salesforce/runner/Runner.java

```diff
@@ -41,6 +41,6 @@ public void beforeAllScenarios() throws PropertiesReadingException {
      */
     @AfterTest
     public void afterAllScenarios() {
-        WebDriverManager.getInstance().quit();
+        WebDriverManager.getInstance().getWebDriver().quit();
```

Comment:
Please, consider use "getInstnace.quit()" instead "getInstance().getWebDriver().quit()" because the first also set the singleton instance as null to the following scenario.